CancerGov/_src/StyleSheets/_tableSort.scss

```diff
@@ -55,62 +55,58 @@ table[data-sortable] td[data-sorted="true"]{
   table[data-sortable] {
     height: 100% !important;
   }
-  
-  table[data-sortable] tfoot tr td:last-child {
-    padding-bottom: 20px;
-    background-color: #ace;
-  }
 
-  table[data-sortable] thead tr th:first-child,
-  table[data-sortable] tbody tr td:first-child
-  /*,
-  table[data-sortable] tfoot tr td:first-child*/ {
-  //background: #ace;
-  top: auto;
-  left: 0.5px;
-  position: absolute !important;
-  width: 160px !important;
-  z-index: 3;
-  }
   table[data-sortable] thead tr th:first-child {
     //background-color: #ace;
     height: 7.85em;
-  }
-  table[data-sortable] tbody tr td:first-child {
-    background-color: #fff;
-    //left: -1px;
-    height: 200px !important;
-    line-height: 1.25em;
+    box-shadow: 0 2px 12px 3px rgba(0,0,0,.2);
   }
 
-  table[data-sortable] tbody tr td, 
-  /*table[data-sortable] tfoot tr td,*/ 
-  table[data-sortable] tr td {
-    height: 200px !important;
-    //padding: 20px;
-    //height: 100% !important;
+  table[data-sortable] thead tr th[data-sorted="false"] {
+    background-color: #2b7bba;
   }
-  table[data-sortable] tfoot tr td {
-    height: auto !important;
+  
+  table[data-sortable] thead tr th:first-child,
+  table[data-sortable] tbody tr td:first-child {
+    //background: #ace;
+    border-right: 1px solid #ccc;
+    border-left: 1.25px solid #ccc;
+    border-bottom: none;
+    top: auto;
+    left: 0px;
+    position: absolute !important;
+    width: 160px !important;
+    z-index: 3;
+  }
+  
+  table[data-sortable] tbody tr:first-child td:nth-child(1) {
+    //background-color: #306;
+    height: calc(100% - 125px) !important;
+    box-shadow: 0 11px 12px 3px rgba(0,0,0,.2);
```

Comment:
Set the y offset here to 0 to prevent shadow from casting over the scrollbar
`box-shadow: 0 0 12px 3px rgba(0,0,0,.2);`

Reply:
removing the y offset (to try to prevent the shadow from casting over the scrollbar) causes the shadow to be cast over the header cell. The height of the fixed column's box shadow was increased to accommodate.